### ISSUE TITLE
[Android Studio] Errors after adding phonegap-push-plugin in capacitor update

### ISSUE THREAD

**Issue by iranicus** [ionic-team/capacitor#440]
Hey folks,
So I recently found out about Capacitor and thought it would be a good idea to start learning it considering it will be replacing ionic later which I currently use for a mobile app project. However I have hit a problem which I have been stuck on all weekend after adding the phonegap-push-plugin to the project, updating capacitor and opening it in Android Studio where it fails on the build with a  "no match found" message on the Configure project:capacitor-android-plugins task in the Apply script build.gradle subtask. Prior to adding the plugin the build went through fine and I had no issues.

**The error:**
`org.gradle.api.ProjectConfigurationException: A problem occurred configuring project ':capacitor-android-plugins'.`
`Caused by: org.gradle.api.GradleScriptException: A problem occurred evaluating project ':capacitor-android-plugins'.`
`Caused by: org.gradle.api.GradleScriptException: A problem occurred evaluating script.`
`Caused by: java.lang.IllegalStateException: No match found
	at java.util.regex.Matcher.group(Matcher.java:536)
	at java_util_regex_MatchResult$group.call(Unknown Source)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:48)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:113)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:125)
	at push_9deomnyt7swihgtmcibnh37x8.doExtractStringFromManifest(X:\XXXX\XXXX\XXXX\XXXXXX\capacitorIonic3Test\ionic-capacitor-app\node_modules\@capacitor\cli\assets\capacitor-android-plugins\gradle-files\phonegap-plugin-push\push.gradle:11)`

**Expected Behavior**
No build issues like when before I added the plugin to the project.

**Steps to reproduce**
1. npm i --save phonegap-push-plugin
2. npm run build
3. npx cap sync
4. Copy the following to android/app/src/main/AndroidManifest.xml (mentioned by capacitor): 
`        <activity android:name="com.adobe.phonegap.push.PushHandlerActivity" android:exported="true" android:permission="${applicationId}.permission.PushHandlerActivity"/>`
        `<receiver android:name="com.adobe.phonegap.push.BackgroundActionButtonHandler"/>`
        `<receiver android:name="com.adobe.phonegap.push.PushDismissedHandler"/>`
        `<service android:name="com.adobe.phonegap.push.FCMService">`
             `<intent-filter>`
            `<action android:name="com.google.firebase.MESSAGING_EVENT"/>`
          `</intent-filter>`
        `</service>`
        `<service android:name="com.adobe.phonegap.push.PushInstanceIDListenerService">`
          `<intent-filter>`
            `<action android:name="com.google.firebase.INSTANCE_ID_EVENT"/>`
          `</intent-filter>`
        `</service>`
  `<uses-permission android:name="android.permission.WAKE_LOCK"/>`
  `<uses-permission android:name="android.permission.VIBRATE"/>`

5. npx cap open android

**Misc information**
```
phonegap-push-plugin: ^2.1.2
npx: 9.7.1
@capacitor/android: 1.0.0-alpha.33
@capacitor/core@1.0.0-alpha.33
@capacitor/cli@1.0.0-alpha.33
```

**System information**
```
    Android SDK Tools : 26.1.1
    Node              : v8.11.1
    npm               : 5.8.0
    OS                : Windows 10

cli packages
    @ionic/cli-utils  : 1.19.2
    ionic (Ionic CLI) : 3.20.0

local packages
    @ionic/app-scripts : 3.1.8
    Cordova Platforms  : none
    Ionic Framework    : ionic-angular 3.9.2
```
**Android Studio Project Structure**
```
Gradle version: 4.1
Android Plugin version: 3.0.1
Android Plugin repository: google(), jcenter
Default Library repository: google(), jcenter
Compile SDK version: API 26: Android 8.0 (Oreo)
```

Any other information you need just let me know, thanks now 👍 .

**Comment by jcesarmobile** [comment]
Fixed in latest release (1.0.0-alpha.34)

BTW, Capacitor is not going to replace Ionic, it's going to complement it.

**Comment by iranicus** [comment]
Cheers, will give it another try. Thanks for the clarification.
